.github/workflows/maven-verify.yml

```diff
@@ -26,4 +26,4 @@ jobs:
     name: Verify
     uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v2
     with:
-      jdk-distribution-matrix: '[ "temurin", "zulu" ]'
+      jdk-distribution-matrix: '[ "temurin", "zulu", "adopt", "liberica", "adopt-openj9" ]'
```

Comment:
* adopt is temurin now
* what is the benefit of benefit? It is still OpenJDK Hotspot. I understand Adopt with OpenJ9 instead of HotSpot.

Reply:
yup. we might need this for javadoc plugin as well